Gemfile.lock

```diff
@@ -143,4 +143,4 @@ DEPENDENCIES
   rspec-rails
 
 BUNDLED WITH
-   1.10.6
+   1.11.2
```

Comment:
Upgrading to bundler 1.12 is a big performance bump according to http://bundler.io/v1.12/whats_new.html